### ISSUE TITLE
Line ending `\n` displayed, rather than used in terminal output

### ISSUE THREAD

**Issue by pomplesiegel** [spark/particle-cli#181]
Issue began following update to `1.8.19` on Mac OS X 10.11.1. 
Sample of output when compiling against photon. There are now no line endings, as interpreted by Terminal. All one big line. 

````
In file included from application.cpp:15:0:\nSerialEvent.h: In function \'void serialEvent()\':\nSerialEvent.h:100:60: error: \'class SystemClass\' has no member named \'version\'\n         Serial.println("System version: " + String( System.version() ) );\n                
````

**Comment by brycekahle** [comment]
Do you know what version you were on before you upgraded?

**Comment by pomplesiegel** [comment]
Sure. 1.8.16

**Comment by pomplesiegel** [comment]
Just uninstalled and reinstalled CLI 1.8.19 to see if the issue persists, and it does.

**Comment by kennethlimcp** [comment]
Here's the related change that caused that: https://github.com/spark/particle-cli/commit/5e5390050b00b358a1ce4671f5dbce0f23f6d384#diff-8444573bf9243299544d0900e0820390L435

**Comment by brycekahle** [comment]
Fixed in 1.8.20

**Comment by pomplesiegel** [comment]
confirmed fixed. thanks!
